pkg/policy/fugue.go

```diff
@@ -44,6 +44,10 @@ func (b *fugueAllowBooleanProcessor) Process(val ast.Value) error {
 	return rego.Bind(val, &b.allow)
 }
 
+func (b *fugueAllowBooleanProcessor) ProcessResource(val ast.Value) error {
+	return nil
```

Comment:
Fugue policy processing strategies are of the single-resource type, but don't need to process the `resources` rule.
Nothing happens for them in case this rule is encountered.